main.js

```diff
@@ -0,0 +1,83 @@
+// // Create object to store data needed for the this branch feature
+const projects = [
+  {
+    name: 'Multi-Post Stories',
+    description: `A daily selection of privately personalized reads; no accounts
+          or sign-ups required. has been the industry's standard dummy
+          text ever since the 1500s, when an unknown printer took a
+          standard dummy text.`,
+    technologies: ['css', 'html', 'bootstrap', 'Ruby'],
+  },
+  {
+    name: 'Multi-Post Stories',
+    description: `A daily selection of privately personalized reads; no accounts
+          or sign-ups required. has been the industry's standard dummy
+          text ever since the 1500s, when an unknown printer took a
+          standard dummy text.`,
+    technologies: ['css', 'html', 'bootstrap', 'Ruby'],
+  },
```

Comment:
- [ ] For each object, ensure to add at least these keys. 
- name, description, tech, image, links

Reply:
I understand that in other templates, they are dynamic but in mine, it is the same and one of the project requirements is to stick to the Figma design as much as possible.
Kindly view my Template using this link, it is template 4 and it has no images and all have the same details and design
https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1